src/project.py

```diff
@@ -218,6 +233,52 @@ def get_tps_time(outfiles):
     return tps, f"{hh:02d}:{mm:02d}:{ss:02d}"
 
 
+@directives(ngpu=1)
+@on_pflow
+@MyProject.operation
+@MyProject.post(rdfed)
+@MyProject.pre(sampled)
+def post_proc(job):
+    import cmeutils
+    from cmeutils.structure import gsd_rdf
+    from cmeutils.structure import get_quaternions
```

Comment:
```suggestion
    from cmeutils.structure import gsd_rdf, get_quaternions
```
we can combine these